### PR TITLE
feat(subscription): provide create/delete of subscriptions

### LINKED ISSUE #34: Provide support for creating and deleting subscriptions
Provide two methods `createSubscription` and `deleteSubscription` that allow to create and delete a OneDrive subscrption.

### PR DESCRIPTION
**Pull request by dominique-pfister**
fix #34 

New instance methods for `OneDrive.js`, not sure whether they'd better be grouped in a separate source file.

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## OneDrive
 Helper class that facilitates accessing one drive.
 
-**Kind**: global class
```
Comment (dominique-pfister) [inline docs/API.md:6]: My editor automatically removes empty trailing space, please tell me if it shouldn't for that file.

**Comment by dominique-pfister** [comment]
@tripodsan can you review?

File: src/OneDrive.d.ts [hunk 2]
```diff
@@ -104,8 +104,12 @@ export declare class OneDrive extends EventEmitter {
 
   listSubscriptions(): Promise<GraphResult>;
 
+  createSubscription(resource: string, notificationUrl: string, clientState: string, changeType: string, expiresIn: number): Promise<GraphResult>;
```
Comment (tripodsan) [inline src/OneDrive.d.ts:107]: this is a bit a long argument list... maybe use an options object instead?

Comment (dominique-pfister) [reply src/OneDrive.d.ts:107]: Well, `resource`, `notificationUrl` and `clientState` are not "optional", so just the latter two arguments, or a `params` object instead?

Comment (tripodsan) [reply src/OneDrive.d.ts:107]: that's what I mean.. 

```js
  createSubscription({
    resource: ..., 
    notificationUrl: ..., 
    clientState: ...., 
    changeType: ...., 
    expiresIn: ...
});

**Pull request merged**